HLTrigger/Configuration/python/Phase2/hltPhase2Setup_cff.py

```diff
@@ -0,0 +1,9996 @@
+    appendToDataLabel = cms.string(''),
+    badStripCuts = cms.PSet(
+        TEC = cms.PSet(
+            maxBad = cms.uint32(4),
+            maxConsecutiveBad = cms.uint32(2)
+        ),
+        TIB = cms.PSet(
+            maxBad = cms.uint32(4),
+            maxConsecutiveBad = cms.uint32(2)
+        ),
+        TID = cms.PSet(
+            maxBad = cms.uint32(4),
+            maxConsecutiveBad = cms.uint32(2)
+        ),
+        TOB = cms.PSet(
+            maxBad = cms.uint32(4),
+            maxConsecutiveBad = cms.uint32(2)
+        )
+    )
+)
+
+MuonDetLayerGeometryESProducer = cms.ESProducer("MuonDetLayerGeometryESProducer")
+
+MuonNumberingInitialization = cms.ESProducer("MuonNumberingInitialization")
+
+MuonTransientTrackingRecHitBuilderESProducer = cms.ESProducer("MuonTransientTrackingRecHitBuilderESProducer",
+    ComponentName = cms.string('MuonRecHitBuilder')
+)
+
+OppositeAnalyticalPropagator = cms.ESProducer("AnalyticalPropagatorESProducer",
+    ComponentName = cms.string('AnalyticalPropagatorOpposite'),
+    MaxDPhi = cms.double(1.6),
+    PropagationDirection = cms.string('oppositeToMomentum')
+)
+
+OppositeAnalyticalPropagatorParabolicMF = cms.ESProducer("AnalyticalPropagatorESProducer",
+    ComponentName = cms.string('AnalyticalPropagatorParabolicMfOpposite'),
+    MaxDPhi = cms.double(1.6),
+    PropagationDirection = cms.string('oppositeToMomentum'),
+    SimpleMagneticField = cms.string('ParabolicMf')
+)
+
+OppositeMaterialPropagator = cms.ESProducer("PropagatorWithMaterialESProducer",
+    ComponentName = cms.string('PropagatorWithMaterialOpposite'),
+    Mass = cms.double(0.105),
+    MaxDPhi = cms.double(1.6),
+    PropagationDirection = cms.string('oppositeToMomentum'),
+    SimpleMagneticField = cms.string(''),
+    ptMin = cms.double(-1.0),
+    useRungeKutta = cms.bool(False)
+)
+
+OppositeMaterialPropagatorParabolicMF = cms.ESProducer("PropagatorWithMaterialESProducer",
+    ComponentName = cms.string('PropagatorWithMaterialParabolicMfOpposite'),
+    Mass = cms.double(0.105),
+    MaxDPhi = cms.double(1.6),
+    PropagationDirection = cms.string('oppositeToMomentum'),
+    SimpleMagneticField = cms.string('ParabolicMf'),
+    ptMin = cms.double(-1.0),
+    useRungeKutta = cms.bool(False)
+)
+
+ParabolicParametrizedMagneticFieldProducer = cms.ESProducer("AutoParametrizedMagneticFieldProducer",
+    label = cms.untracked.string('ParabolicMf'),
+    valueOverride = cms.int32(-1),
+    version = cms.string('Parabolic')
+)
+
+PixelCPEGenericESProducer = cms.ESProducer("PixelCPEGenericESProducer",
+    Alpha2Order = cms.bool(True),
+    ClusterProbComputationFlag = cms.int32(0),
+    ComponentName = cms.string('PixelCPEGeneric'),
+    DoCosmics = cms.bool(False),
+    DoLorentz = cms.bool(False),
+    EdgeClusterErrorX = cms.double(50),
+    EdgeClusterErrorY = cms.double(85),
+    IrradiationBiasCorrection = cms.bool(False),
+    LoadTemplatesFromDB = cms.bool(False),
```

Comment:
This is **NOT** the configuration of the `PixelCPEGenericESProducer` we have agreed upon with @mtosi, see please: https://github.com/cms-sw/cmssw/pull/30465.

What's the reason of using hard-coded errors also in the track fitting part? 

@OzAmram FYI

I think this should use the vanilla `PixelCPEGenericESProducer` configuration for a phase-2 offline reconstruction workflow so:

https://github.com/cms-sw/cmssw/blob/b7ad279596c00a54ffccaf59c79ca20b2dc1507d/RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py#L13-L22

Reply:
Right my mistake, didn't update the cff that I passed to Thiago. The vanilla configuration you pointed out is the one we are currently using (with no hard-coded when track angle is available).

Reply:
thanks for confirming it, please update the config
thanks

Reply:
Ok, to be clear the only difference to be changed was 
```
10c10
<     LoadTemplatesFromDB = cms.bool(True),
---
>     LoadTemplatesFromDB = cms.bool(False),
16c16
<     UseErrorsFromTemplates = cms.bool(True),
---
>     UseErrorsFromTemplates = cms.bool(False),
```

This is done in the next version I am pushing now, where the ESProducers and Sources are split!

Reply:
Thanks @trtomei, excepted that as far as I can see you still have:
```python
LoadTemplatesFromDB = cms.bool(False),
UseErrorsFromTemplates = cms.bool(False),
```
in your current version: [link](https://github.com/cms-sw/cmssw/pull/30817/files#diff-746a05f583aa4b5fb049b0cb72804cedR18) and [link](https://github.com/cms-sw/cmssw/pull/30817/files#diff-746a05f583aa4b5fb049b0cb72804cedR12)
while it should be:

```python
LoadTemplatesFromDB = cms.bool(True),
UseErrorsFromTemplates = cms.bool(True),
```

Reply:
Thanks @mmusich , fixed now for real:
https://github.com/trtomei/cmssw/commit/31c41e2ece32e74976b0d464311a58db49dd6bc4